packages/components/menu/src/menu.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  AnchorController,
+  type EventEmitter,
+  EventsController,
+  type PopoverPosition,
+  RovingTabindexController,
+  event
+} from '@sl-design-system/shared';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import styles from './menu.scss.js';
+import { MenuItemGroup } from './menu-item-group.js';
+import { MenuItem } from './menu-item.js';
+
+export class Menu extends LitElement {
+  /** @private */
+  static override styles: CSSResultGroup = styles;
+
+  /** Controller for managing anchoring. */
+  #anchor = new AnchorController(this, {
+    offset: 4,
```

Comment:
Can you please also add the offset for tooltip, popover and tabs (listbox), everywhere should be `offset 4` as well :)

<img width="224" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/02aa481d-f478-4428-97cd-220b01eb386c">